frontend/src/views/HomeView.vue

```diff
@@ -1 +1,69 @@
-<template><h1>Test</h1></template>
+<script lang="ts" setup>
+import { onMounted, ref } from 'vue'
+import { ElButton, ElDialog, ElInput } from 'element-plus'
+import { useRolesStore } from '@/stores/roles/index'
+import { ErrorMessage } from '@/services/messages/messages'
```

Comment:
You can merge this import, with the below one. If the import come from the same file, you can put in the same line (as element-plus above)